prep-html-skeleton/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE>
+<html>
+  <head>
+    <title>
+      HTML Skeleton Exercise
+    </title>
+    <body>
```

Comment:
Oops! The `body` element needs to be a sibling element to the `head` element, not a child of it.
```
<html>
  <head></head>
  <body></body>
</html>
```